ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -119,6 +134,43 @@ public void SetPointerEvents(BorderedCanvas view, string pointerEventsValue)
             view.SetPointerEvents(pointerEvents);
         }
 
+        #endregion
+
+        #region borders and background
+
+        private class BorderProps
+        {
+            public uint? Color;
+            public double[] Width = new double[4] { 0, 0, 0, 0 };
+        }
```

Comment:
I don't see where `Width` is being used?

Reply:
Fixed